nvtabular/ops/categorify.py

```diff
@@ -360,13 +363,13 @@ def process_vocabs(self, vocabs):
 
     def _create_fit_options_from_columns(self, columns) -> "FitOptions":
         return FitOptions(
-            columns,
-            [],
-            [],
-            self.out_path,
-            self.freq_threshold,
-            self.tree_width,
-            self.on_host,
+            col_groups=columns,
+            agg_cols=[],
+            agg_list=[],
+            out_path=self.out_path,
+            freq_limit=self.freq_threshold,
+            tree_width=self.tree_width,
+            on_host=self.on_host,
             concat_groups=self.encode_type == "joint",
```

Comment:
Whats the reasoning behind the kwargs shift?

Reply:
It's a bit more robust against changes of the class itself. In these case the values we use for these arguments are pretty explanatory but there some other parts of the codebase where that's not the case. So to keep things consistent I changed it here as well.